core/vm/evm.libevm_test.go

```diff
@@ -47,6 +48,10 @@ func (o *evmArgOverrider) OverrideEVMResetArgs(r params.Rules, _ *EVMResetArgs)
 	}
 }
 
+func (o *evmArgOverrider) PreprocessingGasCharge(common.Hash) (uint64, error) {
```

Comment:
nit unless it violates libevm style
```suggestion
func (*evmArgOverrider) PreprocessingGasCharge(common.Hash) (uint64, error) {
```